src/filters/flip.ts

```diff
@@ -0,0 +1,42 @@
+import { IJS } from '../IJS';
+import { copyAlpha } from '../operations';
+import { getOutputImage } from '../utils/getOutputImage';
+
+import flipX from './flipX';
+import flipY from './flipY';
+
+export interface FlipOptions {
+  /**
+   * Image to which the resulting image has to be put.
+   *
+   * @default 'horizontal'
+   */
+  axis?: 'horizontal' | 'vertical' | 'both';
+  /**
+   * Image to which the resulting image has to be put.
+   */
+  out?: IJS;
+}
+
+/**
+ *
+ * Apply a flip filter to an image.
+ *
+ * @param image - Image to process.
+ * @param options - Flip options.
+ * @returns - The processed image.
+ */
+export function flip(image: IJS, options: FlipOptions = {}): IJS {
+  const { axis = 'horizontal' } = options;
+  let newImage = getOutputImage(image, options, { clone: true });
+  if (image.alpha) {
+    copyAlpha(image, newImage);
+  }
```

Comment:
I think alpha is also copied with `clone: true`

Reply:
fixed thnx